dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -207,13 +207,16 @@ public class ConnectorArguments extends DefaultArguments {
   // private final OptionSpec<String> optionDatabase = parser.accepts("database", "database (can be
   // repeated; all if not
   // specified)").withRequiredArg().describedAs("my_dbname").withValuesSeparatedBy(',');
-  private final OptionSpec<File> optionOutput =
+  private final OptionSpec<String> optionOutput =
       parser
           .accepts(
               "output",
-              "Output file or directory name. If the file name, along with the `.zip` extension, is not provided dumper will attempt to create the zip file with the default file name in the directory.")
+              "Output file, directory name, or GCS path. If the file name, along with "
+                  + "the `.zip` extension, is not provided dumper will attempt to create the zip "
+                  + "file with the default file name in the directory. To use GCS, use the format "
+                  + "gs://<BUCKET>/<PATH>.")
```

Comment:
Should we mention that using GCS requires credentials? And maybe a link in the README/Docs?

Reply:
Done